aliveim/aliveim/settings.py

```diff
@@ -19,7 +19,7 @@
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '7aas+1hlykcsjxgr7nvri$+2tamkk2345h#s4gkb2%@uvy-0j$'
+SECRET_KEY = ''
```

Comment:
No problem with this change, but I would have changed once in production :)
While it runs on our local machines it could be even 'abcd1234'. Merging anyway.